src/SimpleGraphs/generators/staticgraphs.jl

```diff
@@ -763,3 +763,25 @@ function lollipop_graph(n1::T, n2::T) where {T <: Integer}
     add_edge!(g, n1, n1+1)
     return g
 end
+
+"""
+    friendship_graph(n)
+Create a [friendship graph](https://en.wikipedia.org/wiki/Friendship_graph) 
+consisting of n copies of the cycle graph C3 with a common vertex`.
+"""
```

Comment:
I think it is good, that you ensure in your algorithm, that the common vertex is the one with id 1, but you should also document this, as otherwise a programmer cannot relay on that fact.